.github/workflows/e2e-dev.yml

```diff
@@ -0,0 +1,73 @@
+name: 'E2E - Development'
+
+on:
+    pull_request:
+        types: ['opened', 'labeled', 'reopened', 'synchronize']
+
+env:
+    CI: true
+    DEV_URL: 'https://test.e2e.dhis2.org/analytics-dev'
+
+concurrency:
+    group: ${{ github.workflow}}-${{ github.ref }}
+    cancel-in-progress: true
+
+defaults:
+    run:
+        shell: bash
+
+jobs:
+    compute-dev-version:
+        if: contains(github.event.pull_request.labels.*.name, 'e2e-record')
```

Comment:
Shouldn't we use a different label than e2e-record to determine whether to run this workflow (here and line 36)? Most of the time I imagine people just want to run the tests without recording since recording hopefully isn't needed most of the time.

Reply:
Yeah sorry, this should have been `e2e dev`. Side note: I should also not have used hyphens in the label names.